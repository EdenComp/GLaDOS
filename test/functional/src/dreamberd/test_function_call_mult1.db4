str breakline = "
";

fn printLn(s) {
    print(s);
    print(breakline);
}

fn main() {
    str a = "Hello World!";
    int b = 10;
    bool c = true;

    printLn(a);
    printLn(b);
    printLn(c);
}

main();