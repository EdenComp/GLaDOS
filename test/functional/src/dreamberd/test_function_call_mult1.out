Hello World!
10
True

